include/irr/core/math/aabb2dSIMDIntBase.h

```diff
@@ -102,7 +103,7 @@ template<typename VECTOR_TYPE>
 class IRR_FORCE_EBO aabb2dSIMD;
 
 template<>
-class aabb2dSIMD<core::vector2df_SIMD> : public aabb2dSIMDBase<aabb2dSIMD, core::vector2df_SIMD>
+class IRR_FORCE_EBO aabb2dSIMD<core::vector2df_SIMD> : public aabb2dSIMDBase<aabb2dSIMD, core::vector2df_SIMD>
```

Comment:
aabb2SIMDBase decl also needs `IRR_FORCE_EBO`

Reply:
i really need to read about it

Reply:
basically put static_asserts that `sizeof(aabb2dSIMDf)==16u` and see what happens